src/UI/worldScreen/TileSidebar.ts

```diff
@@ -88,10 +88,16 @@ export default class TileSidebar implements Page {
         if (project.costs.length == 0) {
             projectHTML.appendChild(UI.makePara("Cost: Free"));
         } else {
-            const cssClass = this.run.inventory.canAfford(project.costs) ? "project-requirement-met" : "project-requirement-unmet";
-            const costDescriptions = project.costs.map((cost: Cost) => `${cost.toString()}`);
-            const costsString = "Cost: " + costDescriptions.join(", ");
```

Comment:
since it's no longer one string, please add a para label that says `"Costs:"` above the list of costs